### PR TITLE
created a script for training the traditional model and a notebook

### PR DESCRIPTION
**Pull request by notthattal**
### traditional_model.py
- Preps data for modeling. This includes stripping the text and adding start and end tokens
- Creates a Sequence to Sequence model which consists of an LSTM encoder and LSTM decoder with an attention mechanism 
- Creates the pytorch dataset needed for training and the training function
- Creates a beam-search function for composing model responses during inference

### qa_prep.ipynb
- Does all of the above
- Contains a few extra functions that we can use for testing like a padding functionality made from scratch and a tokenizer that was not pretrained. 
- Mainly useful for other members to see the testing process and other potentially to find other methods of improving the traditional model script

### CODE REVIEW

**Comment by ahmed-boutar** [comment]
Looks good to me! I still wonder whether using beam search is necessary here or not (since LSTMs output are supposed to not be that great lol). I like the structure you have. It will be easy to turn this into a pipeline for the traditional ML. I will work on the evaluation and feed more training data to it. (The attention layer is pretty cool ngl)
